### ISSUE TITLE
"Success" content clashing with 'success' method

### ISSUE THREAD

**Issue by adrexia** [Firesphere/silverstripe-stripeslack#13]
It some environments, the controller's success method is being called from the template (rather than the Success HTML Field being rendered). This is causing Silverstripe to throw a 'no fortemplate method' error. This can be fixed by supplying names that differ in more than just case.

**Comment by Firesphere** [comment]
Is this for SS3 or SS4? I don't think it's SS4 ready yet...

**Comment by adrexia** [comment]
SS3. Though it’s likely an issue anywhere that doesn’t respect case.

**Comment by Firesphere** [comment]
👍 Thanks, just checking if I'm not overlooking something... I might have a look at it on the next hackday, but does renaming `Success` to `isSuccessful` or something solving the problem

**Comment by adrexia** [comment]
I’m currently working around it with a getSuccess method instead. Eg

‘return $this->failover->Success’

But it’s likely easier to rename the controller method. I think that’s what you are proposing, yeah?

**Comment by Firesphere** [comment]
Yeps :)

**Comment by Firesphere** [comment]
Non-breaking backward compatibility though, as it's still on ss.org in it's current state, so it's a bit a dance around glass shards :D

**Comment by Firesphere** [comment]
Any chance you can PR this?
